lib/happo/public/happo-runner.js

```diff
@@ -221,9 +231,9 @@ window.happo = {
     // Set up the initial object that we will mutate in our recursive function.
     var box = {
       bottom: 0,
-      left: 0,
+      left: undefined,
```

Comment:
Instead of adding the wrapper, you could probably use `Infinity` for these values.